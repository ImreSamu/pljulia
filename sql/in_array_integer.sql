CREATE FUNCTION julia_in_array(x INTEGER[])
RETURNS INTEGER AS $$
    1
$$ LANGUAGE pljulia;
SELECT julia_in_array('{1,2,3}'::INTEGER[]);
DROP FUNCTION julia_in_array(x INTEGER[]);
