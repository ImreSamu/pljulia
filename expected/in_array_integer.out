CREATE FUNCTION julia_in_array(x INTEGER[])
RETURNS INTEGER AS $$
    1
$$ LANGUAGE pljulia;
SELECT julia_in_array('{1,2,3}'::INTEGER[]);
 julia_in_array 
----------------
              1
(1 row)

DROP FUNCTION julia_in_array(x INTEGER[]);
